docker/docker-compose.yml

```diff
@@ -1,55 +1,6 @@
 version: '3.3'
 services:
```

Comment:
I think here should be placed the main part of docker compose - your services (ui, post and so on)